guide/getting-started/software.md

```diff
@@ -51,26 +51,23 @@ Primarily used by developers, Bitcoin CLIs offer feature-rich ways to interact w
    height = 350
 %}
 
-## Block explorers
+## Explorers
 
-<div class="center" markdown="1">
+If the blockchain is a public database, explorers are simply windows into that database.
+
+For Bitcoin, so-called block explorers let users view data about specific transactions, latest blocks, block height, and so on. They also provide insight into bigger picture activity on the Bitcoin network, such as daily transaction numbers. For example, there are typically fewer transactions on weekends, which results in lower fees, which are ideal for low-priority transactions.
+
+As transactions in Lightning payment channels are not recorded on the blockchain, there are also Lightning network explorers. These let you see public information about the network, such as channel count, capacity and status. Only participant nodes can look up specific transaction information in a channel.
```

Comment:
```suggestion
As transactions in Lightning payment channels are not recorded on the blockchain, there are also Lightning network explorers. These let you see public information about the nodes and the network, such as channel count, capacity and status. Only participant nodes can look up specific transaction information in a channel.
```

It would be handy to add some details about explorers like Amboss.space which allow users to 'register' their nodes and get extra benefits from doing so (such as having a page with social contacts). Not sure if you would class lightning.terminal as a explorer but if you do you could also say that they can give users insights into their own nodes so that they have better manage them.